packages/core/src/components/toggletip/toggletip.tsx

```diff
@@ -78,26 +121,45 @@ export class Toggletip {
   render() {
     return (
       <>
-        <dso-info-button
-          aria-describedby="toggle"
-          onClick={this.click}
-          label={this.label}
-          active={this.active}
-          secondary={this.secondary}
-          ref={(element) => (this.infoButton = element)}
-        />
-        <dso-tooltip
-          stateless
-          descriptive
-          id="toggle"
-          strategy="absolute"
-          active={this.active}
-          position={this.position}
-          small={this.small}
-        >
+        <div class="toggletip-container" onClick={this.click} ref={(element) => (this.containerElement = element)}>
+          {["toggle", "secondary"].includes(this.mode) && (
+            <dso-info-button
+              aria-describedby="tooltip"
+              label={this.label}
+              active={this.active}
+              secondary={this.mode === "secondary"}
+              ref={(element) => (this.infoButton = element)}
+            />
+          )}
+          {this.mode === "badge" && <dso-badge status={this.badgeStatus}>{this.label}</dso-badge>}
+          {this.mode === "icon" && <dso-icon icon={this.active ? this.iconActive : this.icon}></dso-icon>}
```

Comment:
Deze opzet is op dezelfde manier ontoegankelijk zoals die nu in de Viewer is opgezet: Immers, een badge is een niet-interactief element waardoor deze niet focusbaar is (en 'n reeks andere problemen)

Hetzelfde geldt voor Icon